sldtk/sldtk.py

```diff
@@ -41,8 +41,7 @@
 def main():
     args = parse_input(config)
     paths = generate_output_paths(args)
-
-    image = cv2.imread(args['image'])
+    image = cv2.imread(args['image'], -1)
```

Comment:
Please use explicit constants instead of integer values for flags, i.e. `cv2.IMREAD_UNCHANGED` in this case.